### DISCUSSION TITLE
Help required on Python Intellisense

### DISCUSSION THREAD

**Discussion by KephaSher** [microsoft/vscode-discussions#1011] [Extension Development QnA] [answered]
My Python Intellisense just stopped working recently, and I'm out of options.
 
I have VSCode version 1.85.1, and am running on macOS version 12.1.
I've downloaded the Microsoft Python Intellisense (Pylance) extension, and I have my interpreter set to the same one as my terminal without a virtual environment.

Any help is appreciated.

**Comment by gjsjohnmurray** [comment, marked as answer]
This is not the right place to raise your issue. Please see the [READ FIRST](https://github.com/microsoft/vscode-discussions/discussions/5) article that is pinned at the top of this forum. It explains the focus of the forum and points to other places. In your case I recommend you raise the issue on the extension repository.

**Reply by KephaSher** [reply]
Got it. Sorry for the inconvenience.
